### DISCUSSION TITLE
Access a backup of a different browser?

### DISCUSSION THREAD

**Discussion by prosvade1337** [moonD4rk/HackBrowserData#329] [Q&A]
So I work at a tech shop and customer wants all the passwords from his old browser data (backup from old computer) but cant just copy over. Is there a way to just look at a backup?

**Comment by prosvade1337** [comment]
Its a chrome browser backup

**Reply by moonD4rk** [reply]
This is a good idea. I have opened an issue and this feature will be supported in the future.

https://github.com/moonD4rk/HackBrowserData/issues/330
